README.md

```diff
@@ -41,13 +41,32 @@ Formats the token amount.
 #### Parameters
 
 - `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
 - `options.digits`: the number of digits to display. Defaults to `2`.
 - `options.displaySign`: whether the sign (`-` or `+`) should be displayed for the amount.
 
 ### TokenAmount#format(options)
 
 Alias to TokenAmount#format().
 
+### TokenAmount#convert(rate, decimals, options)
+
+Converts from a rate, returning a new `TokenAmount` instance with the desired decimals and set
+options.
+
+#### Parameters
+
+- `rate`: the rate to convert from, as a `BigInt`, `String`, `Number` or `TokenAmount`.
+- `decimals`: the amount of decimals, as a `BigInt`, `String`, `Number` or `BigInt`-like (e.g. BN.js).
+- `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
```

Comment:
What do these options do in this case? The `TokenAmount` object doesn't include these internally, right?

Reply:
The symbol, yes; but definitely the commify and the other options below shouldn't be there. :)